excel/src/main/java/org/apache/metamodel/excel/ExcelUtils.java

```diff
@@ -222,7 +222,62 @@ public static String getCellValue(Workbook wb, Cell cell) {
             } else {
                 // TODO: Consider not formatting it, but simple using
```

Comment:
Now with the new methodology implemented for converting the cell to String, I think you should also remove this "TODO".

Reply:
done